.github/component_owners.yml

```
@@ -17,67 +21,101 @@ components:
     - legendecas
   packages/opentelemetry-id-generator-aws-xray:
     - carolabadeer
+  packages/opentelemetry-propagation-utils: []
+    # Unmaintained?
```

Think @open-telemetry/javascript-maintainers or some subset should probably take this

```suggestion
  packages/opentelemetry-propagation-utils:
    - dyladan
    - pichlermarc
    - legendecas
    - blumamir
```

---

Yep, I agree, I'm not to familiar with the package, but it makes sense for us to take it. I'll get familiar with it in time. :+1: